### DISCUSSION TITLE
Support for 2FactorAuth when you have to enter digits on phone

### DISCUSSION THREAD

**Discussion by XRayAdamo** [stratumauth/app#782] [General]
Is there is a support for 2FactorAuth when you have to enter digits on phone? Our company switched something on Microsoft side and now I have to enter number in Microsoft Auth application like this.
<img width="447" alt="image" src="https://github.com/jamie-mh/AuthenticatorPro/assets/4430621/ebf62090-3a10-41a8-9a57-1577cf48e8c5">
Sorry but I don't know what type of Authentication this is.
Since Microsoft removed support for wearable devices from their app, I am looking for something different

**Comment by jamie-mh** [comment]
Hi,

This was discussed in #714. This is a proprietary system from Microsoft and thus is not supported.

Cheers

**Comment by XRayAdamo** [comment]
Thank you
